src/marqo/tensor_search/on_start_script.py

```diff
@@ -259,7 +258,34 @@ def run(self):
         for message in messages:
             self.logger.info(message)
         self.logger.info("completed prewarming patch models")
-            
+
+
+class CheckNLTKTokenizers:
+    """Check if NLTK tokenizers are available, if not, download them.
+
+    NLTK tokenizers are included in the base-image, we do a sanity check to ensure they are available.
+    """
+    def run(self):
+        try:
+            nltk.data.find("tokenizers/punkt")
+        except LookupError:
+            logger.info("NLTK punkt tokenizer not found. Downloading...")
+            nltk.download("punkt")
```

Comment:
In what case do we need to download it given we've already downloaded them in the base image?

Reply:
E.g., starting Marqo locally via `uvicorn`. Generally, we don't need to download it again since they are in the base-image.